about.html

```diff
@@ -0,0 +1,134 @@
+
+    <div class="imgwrapper justify-content-center">
+        <div class="singleimg">
+            <div class="imgabout p-0" style="width: 100%;">
+                <img src="./assets/images/past-event-images/imager.jpg" alt="about image 1" width="500" height="400">
+
+                <div class="redmask">
+                    <div>
+                        <h6 class='ps-4' style="color: rgb(255, 255, 255);font-weight: lighter;">
+                            3rd May 2023
+                        </h6>
+                        <p style="color: white;font-weight: bolder;font-size: 28px;">Helton Hotel</p>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="singleimg">
+            <div class="imgabout p-0" style="width: 100%;">
+                <img src="./assets/images/past-event-images/musium.jpg" alt="about image 2" width="500" height="400">
+
+                <div class="redmask">
+                    <div>
+                        <h6 class='ps-5' style="color: white;">
+                            2nd June 2023
+                        </h6>
+                        <p style="color: white;font-weight: bolder;font-size: 28px;">Sheraten Hotel</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <section id="sponsor" class="partners  text-center dark-bg-color text-secondary-white"
+                    style="background-color:#272a31; margin-bottom: 20px; padding-bottom: 50px;">
+        <h4 class="text-center fs-2 mt-5" style="color: rgb(255, 255, 255);">Partners</h4>
+        <hr class='mb-4' style="color:rgb(255, 51, 0,1) ;border: 2px solid rgb(255, 51, 0,1);width: 10%; margin-left: 45%;">
+
+        <div class="d-flex justify-content-center group2">
+                <div class="partners-logo logos-img">
+                  <a href="https://www.mozilla.org/en-US/firefox/"><img src="./assets/images/partners-images/mozilla 1.png" alt="mozilla"></a>
+                  <a href="https://www.google.com/"><img src="./assets/images/partners-images/googlekorea 1.png" alt="google"></a>
+                  <a href="https://www.naver.com/"><img src="./assets/images/partners-images/naver 1.png" alt="naver"></a>
+                </div>
+                <div class="partners-logo logos-img">
+                  <a href="http://www.kakaocorp.com/"><img src="./assets/images/partners-images/daumkakaologo 1.png" alt="daumkakao"></a>
+                  <a href="https://www.airbnb.com/"><img src="./assets/images/partners-images/airbnb 1.png" alt="iairbnb"></a>
+                </div>
+        </div>
+    </section>
+
+    <footer class="d-flex footer">
+        <div id="all-rights-logo-mobile"><img src="./assets/icons/logo.png" alt="cc korea icon"></div>
+       <div id="all-rights-info">
+        <div id="all-rights-intro"><p>2023 Creative Commons Ethiopia. Some Rights Reserved.</p>
+        </div>
+    </footer>
```

Comment:
- [ ] According to the design guideline the footer  should contain a logo aligned to the left and text aligned to the right and entire content center aligned. Also for mobile size It should have a white background and for desktop size , `About Page`:  should have a dark background white text and `Index page`: should have a white background dark text. So please make sure to implement that.

Your design| Cindy Shin's design
|---|---|
Index page desktop :|
|<img width="1287" alt="Screenshot 2023-06-30 at 9 34 54 AM" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/44798044/2b71367c-7b36-4aff-87e2-f920091d97af">|<img width="847" alt="Screenshot 2023-06-30 at 9 37 02 AM" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/44798044/49177a83-8e8b-43ba-a893-229c4e54c6e3">|
About page desktop :|
|<img width="1286" alt="Screenshot 2023-06-30 at 9 38 10 AM" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/44798044/238c8568-4891-444b-9c33-c6903f2206d7">|<img width="918" alt="Screenshot 2023-06-30 at 9 20 23 AM" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/44798044/56193417-dc77-4fe6-9f60-8a72351e02e3">